plugins/csv/help.md

```diff
@@ -165,14 +165,14 @@ Field numbers (e.g. `f1`) and a range of fields (e.g. `f5-7`) are used to define
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
 |csv|string|None|True|CSV string|None|column1, column2\nvalue1, value2|
```

Comment:
If this can be updated in the plugin.spec.yaml file and here to include the 'csv' example from the new below Example input section.